### PR TITLE
WIP: Added csv from Localite digitization

### PR DESCRIPTION
**Pull request by alexrockhill**
#### What does this implement/fix?
This PR adds read_dig_montage for a csv file, which is the output of the Localite PinPoint EEG software system.  I have also re-added bvct back to the same function from older version which was deleted for some reason.

#### Additional information
I would like to write tests for both this and bvct, but I am having a hard time because there are not example files of these. As they are XML and csv files and only a few KB I would suggest adding one of each to the test files. What would be great is if this PR could be merged first so that I don't have issues with overwriting the read_dig_montage function I need when I go to pull the latest MNE updates.

### CODE REVIEW

**Comment by larsoner** [comment]
> I would like to write tests for both this and bvct, but I am having a hard time because there are not example files of these. As they are XML and csv files and only a few KB I would suggest adding one of each to the test files.

You can add them to `mne-testing-data` (see [README there](https://github.com/mne-tools/mne-testing-data#add-or-change-files-in-the-repo-for-use-with-mne)) then update this PR with your tests

**Comment by alexrockhill** [comment]
Ok will do, right now Localite only exports the electrode positions and not the extra points or event the fiducials so I am talking to them and as soon as I get that sorted out I will work on the files and adding the tests.

File: mne/channels/montage.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
 import os.path as op
 
 import numpy as np
+from pandas import read_csv
```
Comment (agramfort) [inline mne/channels/montage.py]: you cannot do this. pandas is not a hard dependency of MNE. See how we do this in other places in the code.

Comment (drammock) [reply mne/channels/montage.py]: better yet, see this section of the contributor guide: https://mne-tools.github.io/dev/install/contributing.html#importing  :)

File: mne/channels/montage.py [hunk 2]
```diff
@@ -555,8 +556,8 @@ def _check_frame(d, frame_str):
 
 
 def read_dig_montage(hsp=None, hpi=None, elp=None, point_names=None,
-                     unit='auto', fif=None, egi=None, transform=True,
-                     dev_head_t=False):
+                     unit='auto', fif=None, egi=None, bvct=None, csv=None,
+                     transform=True, dev_head_t=False):
```
Comment (massich) [inline mne/channels/montage.py]: Adding parameters like this would break backward compatibility.

**Comment by agramfort** [comment]
@alexrockhill you should not open PRs from your master branch

let us know when you would like us to review.

**Comment by alexrockhill** [comment]
My mistake, I was syncing my master branch with upstream and I messed it up! I am still working on best way to perform the digitization with the neuronavigation company, Localite, and I will update to the branch appropriately for that and fixing the import read_csv and tests once I figure that out.

File: mne/channels/montage.py [hunk 3]
```diff
@@ -718,6 +723,44 @@ def read_dig_montage(hsp=None, hpi=None, elp=None, point_names=None,
...
+
+        root = ElementTree.parse(bvct).getroot()
+        sensors = root.find('CapTrakElectrodeList')
+
+        fids = {}
+        dig_ch_pos = {}
+
+        fid_name_map = {'Nasion': 'nasion','RPA': 'rpa','LPA': 'lpa'}
+
+        scale = dict(mm=1e-3, cm=1e-2, auto=1e-3, m=1)
```
Comment (sappelhoff) [inline mne/channels/montage.py]: You seem to assume here, that the CapTrack device measures electrode locations in units of meters. However, according to the "help" in the CapTrack computer, the distances are specified in millimeters (mm).

Unfortunately, this is not specified in the manual. But I happen to have a captrack device and I checked. I also made an entry for this in the [fieldtrip wiki](http://www.fieldtriptoolbox.org/faq/how_are_the_different_head_and_mri_coordinate_systems_defined/#details-of-the-captrack-coordinate-system)

**Comment by sappelhoff** [comment]
@alexrockhill if you want I can take over and finish this PR for the CapTrack files. I would like to have access to that functionality to play around with EEG DigMontages and perhaps figuring out supporting coordinate systems in MNE-BIDS for EEG data.

**Comment by jasmainak** [comment]
Feel free to take over if it's fast for you. I think @alexrockhill is officially on holiday now ;-)

File: mne/channels/montage.py (hunk 3 above)
Comment (sappelhoff) [reply mne/channels/montage.py]: Ah okay, I see now, this is right. We always scale to meters ... and with `'auto'`, we assume mm :+1:

**Comment by massich** [comment]
@sappelhoff this should be closed shouldn't it? that's what #6663 implemented. If not, feel free to repoen.

**Comment by sappelhoff** [comment]
I had a feeling @alexrockhill wanted to implement two things here ... CapTrak was only one of them --> see the title: "Added csv from Localite digitization"

I don't know what Localite is.

**Pull request closed without merging**